lighthouse-core/runner.js

```diff
@@ -29,7 +29,7 @@ class Runner {
   /**
    * @template {LH.Config.Config | LH.Config.FRConfig} TConfig
    * @param {(runnerData: {requestedUrl: string, config: TConfig, driverMock?: Driver}) => Promise<LH.Artifacts>} gatherFn
-   * @param {{config: TConfig, computedCache: Map<string, ArbitraryEqualityMap>, url?: string, driverMock?: Driver}} runOpts
+   * @param {{config: TConfig, computedCache: Map<string, ArbitraryEqualityMap>, url?: string, driverMock?: Driver, gatherMode?: LH.Gatherer.GatherMode}} runOpts
```

Comment:
why add `gatherMode` to options?

Reply:
TBH I can't remember why I did it this way, we can just pull `gatherMode` off `artifacts.GatherMode`.